onnxscript/values.py

```diff
@@ -272,12 +300,93 @@ def __init__(
         self.source = source
         self.kwargs = kwargs
         self._param_schemas: Optional[tuple[ParamSchema, ...]] = None
+        self._opschema: Optional[onnx.defs.OpSchema] = None
 
     @property
     def name(self):
         """Returns the function name."""
         return self.opname
 
+    @property
+    def opschema(self) -> Optional[onnx.defs.OpSchema]:
```

Comment:
Why we want register schema whenever it's called? Why not register it once?

Reply:
I used a _opschema field to cache the result so it is not recomputed every time `opschema` is accessed. Was I missing something?